components/image-card/image-card.php

```diff
@@ -444,6 +459,15 @@ public function form( $instance ) {
             $link
         );
 
+        // Image Size.
+        $this->field_text(
```

Comment:
We also have `mm_get_image_sizes()` if you want to make this a select dropdown. You can always filter a blank option into the sizes from this component file if you wanted one. :)